src/metrics.rs

```diff
@@ -306,6 +307,17 @@ impl<'a> FuncSpace<'a> {
         color!(stdout, White);
         writeln!(stdout, "{}", val)
     }
+
+    fn dump_json(&self, path: &PathBuf, output_path: &PathBuf) -> std::io::Result<()> {
+        let json_data = serde_json::to_string_pretty(&self).unwrap();
```

Comment:
Could you file a bug to add an option `--pretty` to choose if the dump should be pretty or not ?

Reply:
Sure! :)

Reply:
Done